File: Hash Table/706. Define HashMap/hashmap.cpp
class MyHashMap
{
public:
    vector<int> hashmap;
    int size;

    MyHashMap()
    {
        size = 1000001;
        hashmap.resize(size);
        fill(hashmap.begin(), hashmap.end(), -1);
    }

    void put(int key, int value)
    {
        hashmap[key] = value;
    }

    int get(int key)
    {
        return hashmap[key];
    }

    void remove(int key)
    {
        hashmap[key] = -1;
    }
};

File: Hash Table/706. Define HashMap/hashmap.py
class MyHashMap(object):
    def __init__(self):
        self.size = 1000001
        self.hashmap = [-1] * self.size

    def put(self, key, value):
        """
        :type key: int
        :type value: int
        :rtype: None
        """
        self.hashmap[key] = value

    def get(self, key):
        """
        :type key: int
        :rtype: int
        """
        return self.hashmap[key]

    def remove(self, key):
        """
        :type key: int
        :rtype: None
        """
        self.hashmap[key] = -1
